operations/observability/mixins/workspace/rules/central/workspaces.yaml

```diff
@@ -14,19 +14,17 @@ spec:
 
   - name: workspace-alerts
     rules:
-    - alert: GitpodWorkspaceStuckOnStarting
```

Comment:
should we still keep this alert with e.g. a higher threshold? If all new workspaces suddenly get stuck on starting would we still get an alert if we remove this one?

Reply:
I think we'd be covered by the[ RegularNotActive alert](https://github.com/gitpod-io/gitpod/blob/86697dace803e3acf86c52721bbe07b6c587edcb/operations/observability/mixins/workspace/rules/satellite/workspaces.yaml#L20-L33), which is a derived metric [here](https://github.com/gitpod-io/gitpod/blob/86697dace803e3acf86c52721bbe07b6c587edcb/operations/observability/mixins/workspace/rules/satellite/workspaces.yaml#L14-L18), that'll alert on workspaces that are having trouble starting or stopping (so not active).

Reply:
Let me double check how [this](https://github.com/gitpod-io/gitpod/blob/86697dace803e3acf86c52721bbe07b6c587edcb/operations/observability/mixins/workspace/rules/satellite/workspaces.yaml#L16) is working and get back to you

Reply:
We should be all set with the RegularNotActive alert. The way `gitpod_ws_manager_workspace_activity_total` works, is that an instance will be true if it's actively being used (so it doesn't timeout). If it's false, that means it hasn't started yet, or can no longer actively being used because it's being backed up. 

Marking as resolved, let me know if you need anything else, @WVerlaek , or if you'd like to reopen this comment because you have questions or want to continue dialog. :+1: